rclcpp/include/rclcpp/utilities.hpp

```diff
@@ -27,7 +27,6 @@
 
 #ifdef ANDROID
 #include <sstream>
-
```

Comment:
nit: There's now an extra line deletion in `utilities.hpp` :P

Reply:
Argh! Fixed.